packages/smart-wallet/test/supports.js

```diff
@@ -117,13 +117,15 @@ export const makeMockTestSpace = async log => {
   produce.zoe.resolve(zoe);
   produce.feeMintAccess.resolve(feeMintAccessP);
 
-  const vatLoader = name => {
+  /** @type {VatLoader} */
+  const vatLoader = async name => {
+    /** @typedef {Awaited<WellKnownVats[typeof name]>} VatRoot */
```

Comment:
inline typedef seems smelly, but I don't have a better solution. Consider naming it `ThisVatRoot` or something else to make clear it's just for this switch.

Reply:
I opted for `ReturnedVat`.